spec/controllers/posts_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+Rails::Controller::Testing.install
+
+RSpec.describe PostsController, type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get '/users/:user_id/posts'
+      expect(response).to be_successful
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the correct template' do
+      get '/users/:user_id/posts'
+      expect(response).to render_template(:index)
+    end
```

Comment:
- [ ] Great job adding Spec tests for Posts controller. As per project requirements, correct view template is rendered for the `show` action and not `index` action as is your case with code snippet below
-   `expect(response).to render_template(:index)`. 

Kindly ensure the `show` action is tested with `describe 'GET /show' do`

![image](https://user-images.githubusercontent.com/62849872/218953007-d2bd47a1-67ab-4bce-b999-43b692043b63.png)